src/image/config.rs

```diff
@@ -401,7 +401,7 @@ pub struct History {
     /// a filesystem diff. It is set to true if this history item
     /// doesn't correspond to an actual layer in the rootfs section
     #[serde(skip_serializing_if = "Option::is_none")]
-    #[getset(get_copy = "pub", set = "pub")]
+    #[getset(get = "pub", set = "pub")]
```

Comment:
`get_copy` is coming from the getset crate: https://github.com/Hoverbear/getset

It means that we do return a `Option<bool>` rather than `&Option<bool>` when calling `empty_layer()` by doing a copy of the member.

Reply:
Ok wow, I had not realized that the derive builder crate is different than getset.  Umm...OK, that took some reverse engineering but I see the bug: we weren't deriving `CopyGetters` and apparently specifying `get_copy` is a no-op without that.

So...I have no strong opinion on which we do.  Do you?

Reply:
Ah, yeah let’s add `CopyGetters` then. I’d love if GetSet would make an auto copy of types which implement the Copy trait. I’d like to avoid working with references to options and such.

Reply:
OK done; I spot checked and only saw one other place that was missing `CopyGetters`.

Reply:
Thank you!